src/injected/content/clipboard.js

```diff
@@ -1,29 +1,17 @@
-let textarea;
 let clipboardData;
 
-function init() {
-  textarea = document.createElement('textarea');
-  textarea.style.position = 'absolute';
-  textarea.style.width = 0;
-  textarea.style.height = 0;
-  textarea.style.left = '-20px';
-}
-
 function onCopy(e) {
+  document.removeEventListener("copy", onCopy, true);
+  e.stopImmediatePropagation();
```

Comment:
This is the key, I think. Just adding this line and removing the `<textarea>` should make it work as expected.